tests/api/server.py

```diff
@@ -8,7 +8,7 @@
 import autograder.api.constants
 import autograder.util.timestamp
 
-PORT = 12345
+PORT = 8080
```

Comment:
Why this change?

I think the static port does indeed need to be changed,
but to a dynamic port.
(Where we just randomly choose any non-well-known port, see if it is open, and pick again if it is not open.)

So for this PR, you can just change it back or do the dynamic selection (either is fine).